### PR TITLE
Move to lib and get import working

### PR DESCRIPTION
**Pull request by jrschumacher**
changes: _rev_

### Proposed Changes
_Please use the Jira Key or NOREF followd by the changes_

- JIRAKEY-1234
  - Some change
  - Another change

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [ ] I have verified that my changes have not introduced new lint errors
- [ ] I have updated the change log

### CODE REVIEW

**Comment by avkvirtru** [comment]
new Layout component doesn't seem to be rendering for me locally… or is this a WIP?
image

File: package.json [hunk 1]
```diff
@@ -4,9 +4,18 @@
   "description": "Thanks for using this template! 🙏 The goal of this template is to simplify the creation of repositories and encourage Virtru best practices.",
   "main": "index.js",
```
Comment (avkvirtru) [inline package.json:5]: Should this point to `lib/index.js`?

**Comment by jrschumacher** [comment]
I think I messed something up when I moved it to lib. Let me look at it again.

File: lib/helpers/useIsMobile.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import useMatchMedia from './useMatchMedia';
+
+export default function useIsMobile() {
+  useMatchMedia('screen and (max-width: 767px)');
```
[the remark this answers is not in the document]
Comment (jrschumacher) [reply lib/helpers/useIsMobile.js:4]: @avkvirtru we need to figure out how we can have the variables in JavaScript and css variables then we can do this.

File: lib/styles/tokens.css [hunk 3]
```diff
@@ -167,3 +169,22 @@
   --vds-line-height-heading-lg:   48px;
   --vds-line-height-heading-xl:   54px;
 }
+
+
+/**
+ * @tokens Container Widths
+ * @presenter ContainerWidth
```
Comment (avkvirtru) [inline lib/styles/tokens.css:176]: FYI these are limited to this list of [presenters](https://github.com/UX-and-I/storybook-design-token#presenters).

**Pull request merged**